pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -48,9 +48,9 @@ func init() {
 }
 
 type internalConfig struct {
-	limitCPUToMemoryRatio     *inf.Dec
-	cpuRequestToLimitRatio    *inf.Dec
-	memoryRequestToLimitRatio *inf.Dec
+	limitCPUToMemoryRatio     float64
```

Comment:
I don't really have a problem with switching to float but I thought the aim of the Resource and Quantity libs was to avoid that. Is it basically impossible to avoid float multiplication against the latest implementation?